src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Program.Main.cs

```diff
@@ -153,6 +153,11 @@ public static void Main(string[] args)
 
         app.UseRouting();
 
+        #if (IndividualAuth)
+        app.UseAuthorization();
+
+        #endif
+
```

Comment:
Will this add an extra newline?

Reply:
Yeah, I believe you're right. It didn't previously have it.

https://github.com/dotnet/aspnetcore/blob/9ac46165aa66079292ef32b2fbfebfb8717fd6e1/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Program.Main.cs#L156-L161

```suggestion
```